src/services/web3/web3Reducer.js

```diff
@@ -0,0 +1,20 @@
+const web3Reducer = (state = { web3Instance: null }, action) => {
```

Comment:
For consistency's sake and for general practice, why not define the initial state separately above the reducer definition? Also makes it clearer what is happening if you expand on this later.